### ISSUE TITLE
refactor to follow node async (error, result) api pattern?

### ISSUE THREAD

**Issue by stephenhandley** [mckelvey/instagram-node-lib#26]
for example Instagram.media.subscribe(options) would accept function(error, result) {...} as second argument etc. etc.

**Comment by mckelvey** [comment]
@stephenhandley— I am considering rewriting the library to do just this, e.g. less a jQuery model of success/failure and more the node model that has since become prevalent.

**Comment by stephenhandley** [comment]
awesome, yeah that would great

**Comment by aj0strow** [comment]
Yeah, unfortunately kinda unusable. Callbacks / promises / events please!

**Comment by despairblue** [comment]
:+1:
